src/get/index.js

```diff
@@ -109,13 +109,15 @@ async function get(options) {
     }
 
     /**
-     * If the compressed binary exists, then `true`. Otherwise, it is `false`. 
+     * If the compressed binary exists, then `true`. Otherwise, it is `false`.
      *
      * @type {boolean}
      */
     const ffmpegFilePathExists = await util.fileExists(ffmpegFilePath);
     if (ffmpegFilePathExists === false) {
-      ffmpegFilePath = await ffmpeg(options.downloadUrl, options.version, options.platform, options.arch, options.cacheDir);
+      const downloadUrl =
+        "https://github.com/nwjs-ffmpeg-prebuilt/nwjs-ffmpeg-prebuilt/releases/download";
+      ffmpegFilePath = await ffmpeg(downloadUrl, options.version, options.platform, options.arch, options.cacheDir);
```

Comment:
```suggestion
      options.downloadUrl =
        "https://github.com/nwjs-ffmpeg-prebuilt/nwjs-ffmpeg-prebuilt/releases/download";
      ffmpegFilePath = await ffmpeg(options.downloadUrl, options.version, options.platform, options.arch, options.cacheDir);
```

Reply:
I'm not sure it's necessary to set the downloadurl for ffmpeg onto the global options. Remember this is JUST the ffmpeg download url, not the nwjs download url

Reply:
Aghh true, nice catch! This would probably error out when `options.ffmpeg` _and_ `options.nativeAddon` are enabled.